src/styles/blocks/cal.scss

```diff
@@ -0,0 +1,46 @@
+@import "../utils/variables";
```

Comment:
you have already imported this in `main.scss` do not import it here

Reply:
done